docs/source/api/algorithms/clustering/kmeans-init.rst

```diff
@@ -75,8 +75,8 @@ Operation
               kmeans_init::compute_result compute(const Descriptor& desc, \
                                          const kmeans_init::compute_input& input)
 
-   :tparam desc: K-Means algorithm descriptor :expr:`kmeans_init::desc`
-   :tparam input: Input data for the computing operation
+   :param descriptor: K-Means algorithm descriptor :expr:`kmeans_init::descriptor`
```

Comment:
```suggestion
   :param desc: K-Means algorithm descriptor :expr:`kmeans_init::descriptor`
```

`desc` is name of the argument of the `compute` function above. `descriptor` is a class name. This change should be applied to all algorithms.

Reply:
done